environments/apis.go

```diff
@@ -13,6 +15,10 @@ type Api struct {
 	Endpoint ApiEndpoint
 }
 
+func (a *Api) IsAvailable() bool {
```

Comment:
tbh since Api is always a concrete object we could make this:

```suggestion
func (a Api) IsAvailable() bool {
```

and then remove the a != nil check below?